doc/sphinx/source/notes_6_1.rst

```diff
@@ -39,6 +39,12 @@ Work-group vectorization improvements
 Miscellaneous
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* New env variable POCL_MAX_COMPUTE_UNITS. This is similar to
+  existing POCL_CPU_MAX_CU_COUNT, but is not driver-specific,
+  hence can be used to limit the CU count of all drivers that
+  support limiting CU count. If both are specified, the driver
+  specific variable takes precedence.
+
```

Comment:
Is there use cases for `POCL_CPU_MAX_CU_COUNT`? Could all uses of it replaced with the`POCL_CPU_MAX_COMPUTE_UNITS`? I was thinking if the `POCL_CPU_MAX_CU_COUNT` should be deprecated (document it, emit warning message if it used).

Reply:
I think the use case is exactly that: If we only want to limit the CPU's max compute unit count, but not other drivers. I don't know if this is a realistic use case, and we discussed this locally and came to the conclusion that we have so many envs that it doesn't hurt to keep this one ;)